torchvision/prototype/transforms/_misc.py

```diff
@@ -207,7 +207,9 @@ def _get_params(self, flat_inputs: List[Any]) -> Dict[str, Any]:
         #  format,we need to convert first just to afterwards compute the width and height again, although they were
         #  there in the first place for these formats.
         bounding_box = F.convert_format_bounding_box(
-            bounding_box, old_format=bounding_box.format, new_format=features.BoundingBoxFormat.XYXY
+            bounding_box.as_subclass(torch.Tensor),
```

Comment:
I'm adopting the idiom of doing the `as_subclass` right on the `F` call, so that any other invocations within the implementation on properties of BBoxes are not affected.